### ISSUE TITLE
HeaderTest.php does not comply with psr-4 autoloading standard

### ISSUE THREAD

**Issue by karmendra** [mtibben/html2text#98]
Due to different ClassName and FileName we see following warning while doing `composer dump-autoload`

```
vagrant@homestead:~/code/mrm$ composer dump
Generating optimized autoload files
Deprecation Notice: Class Html2Text\StrToUpperTest located in ./vendor/html2text/html2text/test/HeaderTest.php does not comply with psr-4 autoloading standard. It will not autoload anymore in Composer v2.0. in phar:///usr/local/bin/composer/src/Composer/Autoload/ClassMapGenerator.php:201

```

**Comment by roelofr** [comment]
Id' actually recon that the loading of the tests is wrong in the first place.

The JSON file currenlty looks like this:
```json
{
    // ...
    "autoload": {
        "psr-4": {
            "Html2Text\\": [
                "src/",
                "test/"
            ]
        }
    }
}
```

However, Composer also has an [`autoload-dev` property][1], which should be used for tests instead. The classmap need not be polluted with tests.

```json
{
    // ...
    "autoload": {
        "psr-4": {
            "Html2Text\\": "src/"
        }
    },
    "autoload-dev": {
        "psr-4": {
            "Html2Text\\": "test/"
        }
    }
}
```

[1]: https://getcomposer.org/doc/04-schema.md#autoload-dev

I'm doubtful if @mtibben is still supporting this package, given the high number of stale pull requests (see #99 and #97)

**Comment by mtibben** [comment]
Fixed in #99
